ghostTemplate.yaml

```diff
@@ -0,0 +1,381 @@
+            # https://ghost.org/docs/concepts/config/
+            - Name: database__connection__ssl__ca
+              Value: !If [GhostAuroraDB, "-----BEGIN CERTIFICATE-----\nMIIDQTCCAimgAwIBAgITBmyfz5m/jAo54vB4ikPmljZbyjANBgkqhkiG9w0BAQsF\nADA5MQswCQYDVQQGEwJVUzEPMA0GA1UEChMGQW1hem9uMRkwFwYDVQQDExBBbWF6\nb24gUm9vdCBDQSAxMB4XDTE1MDUyNjAwMDAwMFoXDTM4MDExNzAwMDAwMFowOTEL\nMAkGA1UEBhMCVVMxDzANBgNVBAoTBkFtYXpvbjEZMBcGA1UEAxMQQW1hem9uIFJv\nb3QgQ0EgMTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBALJ4gHHKeNXj\nca9HgFB0fW7Y14h29Jlo91ghYPl0hAEvrAIthtOgQ3pOsqTQNroBvo3bSMgHFzZM\n9O6II8c+6zf1tRn4SWiw3te5djgdYZ6k/oI2peVKVuRF4fn9tBb6dNqcmzU5L/qw\nIFAGbHrQgLKm+a/sRxmPUDgH3KKHOVj4utWp+UhnMJbulHheb4mjUcAwhmahRWa6\nVOujw5H5SNz/0egwLX0tdHA114gk957EWW67c4cX8jJGKLhD+rcdqsq08p8kDi1L\n93FcXmn/6pUCyziKrlA4b9v7LWIbxcceVOF34GfID5yHI9Y/QCB/IIDEgEw+OyQm\njgSubJrIqg0CAwEAAaNCMEAwDwYDVR0TAQH/BAUwAwEB/zAOBgNVHQ8BAf8EBAMC\nAYYwHQYDVR0OBBYEFIQYzIU07LwMlJQuCFmcx7IQTgoIMA0GCSqGSIb3DQEBCwUA\nA4IBAQCY8jdaQZChGsV2USggNiMOruYou6r4lK5IpDB/G/wkjUu0yKGX9rbxenDI\nU5PMCCjjmCXPI6T53iHTfIUJrU6adTrCC2qJeHZERxhlbI1Bjjt/msv0tadQ1wUs\nN+gDS63pYaACbvXy8MWy7Vu33PqUXHeeE6V/Uq2V8viTO96LXFvKWlJbYK8U90vv\no/ufQJVtMVT8QtPHRh8jrdkPSHCa2XV4cdFyQzR1bldZwgJcJmApzyMZFo6IQ6XU\n5MsI+yMRQ+hDKXJioaldXgjUkK642M4UwtBV8ob2xJNDd2ZhwLnoQdeXeGADbkpy\nrqXRfboQnoZsG4q5WTP468SQvvG5\n-----END CERTIFICATE-----\n", !Ref "AWS::NoValue"]
+      Cpu: '256'
+      ExecutionRoleArn: !GetAtt ghostAPIExecutionRole.Arn
+      Memory: '512'
+      NetworkMode: awsvpc
+      RequiresCompatibilities:
+        - FARGATE
+      TaskRoleArn: !If [GhostAuroraDB, !GetAtt ghostAPITaskRoleAurora.Arn, !GetAtt ghostAPITaskRoleEFS.Arn]
+      Volumes: 
+        - Name: ghostVolume
+          EFSVolumeConfiguration:
+            FilesystemId: !GetAtt ghostFS.FileSystemId
+            AuthorizationConfig:
+              IAM: "ENABLED"
+            TransitEncryption: 'ENABLED'
+    DependsOn: ghostAPILogGroup
+  Service:
+    Type: AWS::ECS::Service
+    Properties:
+      ServiceName: !Sub ghostAPIservice-${EnvironmentTagName}
+      PlatformVersion: 1.4.0
+      Cluster: default
+      TaskDefinition: !Ref ghostAPI
+      DesiredCount: 2
+      DeploymentConfiguration:
+        MaximumPercent: 200
+        MinimumHealthyPercent: 100
+      LaunchType: FARGATE
+      NetworkConfiguration:
+        AwsvpcConfiguration:
+          AssignPublicIp: ENABLED
+          Subnets: !Ref DefaultVPCSubnets
+          SecurityGroups:
+            - !GetAtt ghostAPISecurityGroup.GroupId
+      LoadBalancers:
+        - TargetGroupArn: !Ref TargetGroup
+          ContainerPort: 2368
+          ContainerName: 'ghost'    
+  Loadbalancer:
+    Type: AWS::ElasticLoadBalancingV2::LoadBalancer
+    Properties: 
+      IpAddressType: ipv4
+      Name: !Sub ghostAPILB-${EnvironmentTagName}
+      SecurityGroups: 
+        - !GetAtt ghostLBSecurityGroup.GroupId
+      Subnets: !Ref DefaultVPCSubnets
```

Comment:
IIRC, this will only register listeners in subnets where there are targets. If the Fargate service is updated to only run in the first two default VPC subnets, this will end up only registering listeners in those two subnets. But it might not hurt to double check my memory on this. If you see listeners spun up in other subnets that would be a sign this should also be scoped to be an array of only the first two subnets.